napari/layers/labels/labels.py

```diff
@@ -893,6 +894,26 @@ def get_color(self, label):
             col = self.colormap.map(val)[0]
         return col
 
+    def _get_value_ray(self, start_point, end_point):
```

Comment:
Is this going to turn into an abstract method on the base class that every layer will have to implement, sort of like some of the `_get_value`? That makes a lot of sense to me.

If you liked the idea of adding a kwarg `world_to_data` that I mentioned you could do the same thing with 

```python
get_value(self, position, *, world=False, ray=False):
```

which I think would be quite nice. cc @jni